app/degreePlanner/degreePlanner.css

```diff
@@ -9,10 +9,11 @@
 
 .formElement .degreeLabel {
     font-size: 18px;
+    display: flex;
```

Comment:
Why do we need this?

Reply:
I added display: flex; to the .formElement .degreeLabel class to ensure that the label aligns properly above the input field.

Reply:
Unexpected behaviour? When the line is removed, it only changed the first two degree labels but not the others (even though they have the same class name)
<img width="248" alt="Screenshot 2024-05-29 at 6 02 05 PM" src="https://github.com/GoodGameRuler/unitrack_v3/assets/63948459/64cd33bb-5b04-4f45-9ede-823bf3a90403">
<img width="610" alt="Screenshot 2024-05-29 at 6 02 01 PM" src="https://github.com/GoodGameRuler/unitrack_v3/assets/63948459/e01bd72b-c82f-4160-b46b-67465eb18550">